File: cpp/grammer/04_list.cpp
#include <algorithm>
#include <iostream>
#include <vector>

int main() {
    std::vector<int> arr = {3, 1, 2};
    std::cout << "초기 리스트: ";
    for (int x : arr) std::cout << x << ' ';
    std::cout << "\n";

    arr.push_back(4);
    arr.erase(std::find(arr.begin(), arr.end(), 1));
    std::sort(arr.begin(), arr.end());

    std::cout << "변경 후 리스트: ";
    for (int x : arr) std::cout << x << ' ';
    std::cout << "\n";
    std::cout << "첫 번째 원소: " << arr.front() << " 마지막 원소: " << arr.back() << "\n";
}

File: python/grammer/04_list.py
"""4) 리스트: 추가/삭제/정렬 같은 기본 조작."""

arr = [3, 1, 2]
print("초기 리스트:", arr)

arr.append(4)        # 맨 뒤 추가
arr.remove(1)        # 값 1 제거
arr.sort()           # 오름차순 정렬

print("변경 후 리스트:", arr)
print("첫 번째 원소:", arr[0], "마지막 원소:", arr[-1])
